examples/extra_envs/test_install.sh

```diff
@@ -3,21 +3,20 @@ set -ex
 
 # if PREFIX is not defined, then this was called from an OSX PKG installer
 # $2 is the install location, ($HOME by default)
-if [ xxx$PREFIX = 'xxx' ]; then
+if [ -z "${PREFIX:-}" ]; then
```

Comment:
Is the `:-` really necessary (here and below)?

Reply:
It helps with cases where `set -u` has been opted in. Otherwise it can throw unbound errors. We don't strictly need it here, but I guess it doesn't hurt serving a good example in case people copy things around without realizing the caveats. This expansion is POSIX compliant too, so again, I don't think it hurts.

Reply:
Gotcha, thanks for the explanation!